runtime/compiler/aarch64/codegen/ARM64PrivateLinkage.cpp

```diff
@@ -1665,32 +1665,121 @@ static void buildVirtualCall(TR::CodeGenerator *cg, TR::Node *callNode, TR::Regi
 /**
  * @brief Generates instruction sequence for interface call
  *
- * @param[in] cg:          code generator
- * @param[in] callNode:    node for the interface call
- * @param[in] vftReg:      vft register
- * @param[in] tmpReg:      temporary register
- * @param[in] tmp2Reg:     temporary register
- * @param[in] ifcSnippet:  interface call snippet
- * @param[in] regMapForGC: register map for GC
+ * @param[in] cg:                 code generator
+ * @param[in] callNode:           node for the interface call
+ * @param[in] vftReg:             vft register
+ * @param[in] x9Reg:              temporary register
```

Comment:
Please clarify why this temporary register is called as `x9Reg` while other two temporary registers are `tmpReg` and `tmp2Reg`.
If the comparison with the instruction `sub x9, x9, x11` in PicBuilder.spp is the reason, then I guess it is better to rename `tmp2Reg` to `x11Reg`.

Reply:
Agree. I renamed `tmpReg` to `x10Reg` and `tmp2Reg` to `x11Reg`.